doc/Usage.md

```diff
@@ -21,6 +21,130 @@ $text = $pdf->getText();
 $text = $pdf->getPages()[0]->getText();
 ```
 
+## Extract text positions
+
+You can extract transformation matrix (indexes 0-3) and x,y position of text objects (indexes 4,5).
+
+```php
+$data = $pdf->getPages()[0]->getDataTm();
+
+Array
+(
+    [0] => Array
+        (
+            [0] => Array
+                (
+                    [0] => 0.999429
+                    [1] => 0
+                    [2] => 0
+                    [3] => 1
+                    [4] => 201.96
+                    [5] => 720.68
+                )
+
+            [1] => Document title
+        )
+
+    [1] => Array
+        (
+            [0] => Array
+                (
+                    [0] => 0.999402
+                    [1] => 0
+                    [2] => 0
+                    [3] => 1
+                    [4] => 70.8
+                    [5] => 698.96
+                )
+
+            [1] =>
```

Comment:
```suggestion
    [1] => Array
        (
            [0] => Array
                (
                    [0] => 0.999402
                    [1] => 0
                    [2] => 0
                    [3] => 1
                    [4] => 70.8
                    [5] => 698.96
                )

            [1] =>
```

Does it make sense to remove this part, because it is not clear what it shows. Elements of index 0 and 2 are more clear. What do you think?

Reply:
It is only space which is not shown by print_r. I wil remove it.